CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ Please also have a look at our
 
 ### Removed
 
+- Remove `OutputFormat::level()` (#874))
```

Comment:
```suggestion
- Remove `OutputFormat::level()` (#874)
```

Reply:
Damn. I really should read the changelog, too, before pushing.